src/test/java/bio/terra/service/dataset/DatasetSchemaUpdateIntegrationTest.java

```diff
@@ -0,0 +1,174 @@
+package bio.terra.service.dataset;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import bio.terra.common.category.Integration;
+import bio.terra.integration.DataRepoFixtures;
+import bio.terra.integration.TestJobWatcher;
+import bio.terra.integration.UsersBase;
+import bio.terra.model.ColumnModel;
+import bio.terra.model.DatasetModel;
+import bio.terra.model.DatasetSchemaColumnUpdateModel;
+import bio.terra.model.DatasetSchemaUpdateModel;
+import bio.terra.model.DatasetSchemaUpdateModelChanges;
+import bio.terra.model.DatasetSummaryModel;
+import bio.terra.model.TableDataType;
+import bio.terra.model.TableModel;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+// TODO move me to integration dir
```

Comment:
Is this a TODO that can be taken care of now?

Reply:
Hmm I noticed this comment in a few tests, but I'm not sure if it's still relevant. I see we have a top-level integration directory (`tests/integration`). Personally I like the idea of adding a subdirectory for unit, connected and integration tests in `tests/service/<dir>/` instead. I think it makes it easier to all of the tests that exist for a given part of the codebase.